### PR TITLE
Fixed bug in `unlink_account()` of Pallet Accounts

### PR DESCRIPTION
**Pull request by aang114**
Very small PR

### CODE REVIEW

File: pallets/accounts/src/lib.rs [hunk 1]
```diff
@@ -868,8 +868,12 @@ pub mod pallet {
 		/// Unlink the **Clamor public account address `sender`** from **its linked EVM public
 		/// account address `account`**
 		fn unlink_account(sender: T::AccountId, account: H160) -> DispatchResult {
-			ensure!(<EVMLinks<T>>::contains_key(sender.clone()), Error::<T>::AccountNotLinked);
-			ensure!(<EVMLinksReverse<T>>::contains_key(account), Error::<T>::AccountNotLinked);
+			if <EVMLinks<T>>::get(sender.clone()).ok_or(Error::<T>::AccountNotLinked)? != account {
+				return Err(Error::<T>::AccountNotLinked.into());
```
Comment (sinkingsugar) [inline pallets/accounts/src/lib.rs]: Maybe we need a new kind of error?

**Review by sinkingsugar — CHANGES_REQUESTED** [review]

File: pallets/accounts/src/lib.rs (hunk 1 above)
Comment (alebaffa) [reply pallets/accounts/src/lib.rs]: Agree.

**Review by alebaffa — CHANGES_REQUESTED** [review]

File: pallets/accounts/src/lib.rs (hunk 1 above)
Comment (aang114) [reply pallets/accounts/src/lib.rs]: i've added a new kind of error. please let me know if it's okay. thanks

**Review by alebaffa — APPROVED** [review]
LGTM

File: pallets/accounts/src/lib.rs [hunk 2]
```diff
@@ -869,10 +871,10 @@ pub mod pallet {
 		/// account address `account`**
 		fn unlink_account(sender: T::AccountId, account: H160) -> DispatchResult {
 			if <EVMLinks<T>>::get(sender.clone()).ok_or(Error::<T>::AccountNotLinked)? != account {
-				return Err(Error::<T>::AccountNotLinked.into());
+				return Err(Error::<T>::DifferentAccountLinked.into());
```
Comment (sinkingsugar) [inline pallets/accounts/src/lib.rs:874]: How about `AccountAlreadyLinked` 🤔

Comment (aang114) [reply pallets/accounts/src/lib.rs:874]: isn't `DifferentAccountLinked` more accurate?

Comment (sinkingsugar) [reply pallets/accounts/src/lib.rs:874]: I think it's equal , just for consistency with other errors, but doesn't matter really

**Review by sinkingsugar — APPROVED** [review]

File: pallets/accounts/src/lib.rs (hunk 2 above)
Comment (aang114) [reply pallets/accounts/src/lib.rs:874]: okay then i'm just merging it 😅

**Pull request merged**